src/lib/components/MuiGridList/components/MuiGridListTileBar/MuiGridListTileBar.js

```diff
@@ -6,6 +6,9 @@ import {
   validComponents,
   muiStringProp,
 } from '../../../../utils/customProps';
+import MuiIconButton from '../../../MuiIconButton/MuiIconButton';
```

Comment:
i dont think we need the MuiIconButton the only thing that can do is show a tooltip